components/_modals.scss

```diff
@@ -0,0 +1,118 @@
+/*
+
+Stylesheet for all things related to presentation of 
+modals across the website.
+
+*/
+
+// Overlay behind the modal
+
+.ui-widget-overlay {
+	background: $tws-yellow;
+}
+
+// General modal box positioning
+
+.ui-dialog {
+
+	position: fixed !important;
+  top: 50% !important;
+  left: 50% !important;
+  -webkit-transform: translate(-50%, -50%) !important;
+  -moz-transform:    translate(-50%, -50%) !important;
+  -ms-transform:     translate(-50%, -50%) !important;
```

Comment:
!important should not be used<br>Colon after property should be followed by one space<br>Avoid vendor prefixes.